arangod/Replication/TailingSyncer.cpp

```diff
@@ -1189,33 +1192,15 @@ Result TailingSyncer::runInternal() {
       }
 
       // remove previous applier state
-      abortOngoingTransactions();
-
-      _applier->removeState();
+      abortOngoingTransactions(); //ties to clear map - no further side effects
 
-      // TODO: merge with removeState
-      {
-        WRITE_LOCKER_EVENTUAL(writeLocker, _applier->_statusLock);
-
-        LOG_TOPIC(DEBUG, Logger::REPLICATION)
-            << "stopped replication applier for database '" << _state.databaseName
-            << "' with lastProcessedContinuousTick: " << _applier->_state._lastProcessedContinuousTick
-            << ", lastAppliedContinuousTick: " << _applier->_state._lastAppliedContinuousTick
-            << ", safeResumeTick: " << _applier->_state._safeResumeTick;
-
-        _applier->_state._lastProcessedContinuousTick = 0;
-        _applier->_state._lastAppliedContinuousTick = 0;
-        _applier->_state._safeResumeTick = 0;
-        _applier->_state._failedConnects = 0;
-        _applier->_state._totalRequests = 0;
-        _applier->_state._totalFailedConnects = 0;
-        _applier->_state._totalResyncs = 0;
```

Comment:
Not all of these variables are zeroed out by the `ReplicationApplierState::reset` call inside `resetState`, and that call zeroes out some other variables as well. I'm less than 100% sure that we want the different set of variables cleared. Reasoning?